### PR TITLE
Intro

### LINKED ISSUE #47: Set up Introduction.md
Help people get started working on this repo

### PR DESCRIPTION
**Pull request by GaithAST**
# Description

Full introduction done!

Fixes #47 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### CODE REVIEW

File: INTRODUCTION.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+<br />
+
+## Setting the project up locally
+To setup the project locally, we are going to use [Github Desktop](https://desktop.github.com/). So after you've downloaded github desktop, login into your GitHub account. Then press on file (top left) and clone repository. Click on the fork you just made and select the path of where you want to install it in the bottom textfield
+<br />
+<br>
+<br />
+
+## pub get
+after its done cloning the repo, open the folder in vs code or your preferred code editor, and run a ``` flutter pub get``` to get download all the dependencies the project uses.
```
Comment (tadaspetra) [inline INTRODUCTION.md:32]: capitalize :)

File: INTRODUCTION.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+ 
+To enable a feature, press on the feature in the left navbar under the build section, then clicking on 'Get Started' Or 'Create Database' respectively
+
+<br />
+<br> 
+
+### Setting up authentication in firebase
+
+<br>
+after you enable the authentication feature in your firebase project, you're going to enable Google sign in if youre developing on android, and both google and apple if youre developing on ios
```
Comment (tadaspetra) [inline INTRODUCTION.md:111]: capitalize :)

File: INTRODUCTION.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+<br />
+
+## Setting up firebase
+ 
+ To set up firebase, were going to create a new firebae project by going to [firebase console](https://console.firebase.google.com/u/0/) and creating a new project. When we're done creating the firebase project, youre gonna enable the following features:
+ 
+ * Firestore Database
+ * Authentication
+ 
+To enable a feature, press on the feature in the left navbar under the build section, then clicking on 'Get Started' Or 'Create Database' respectively
```
Comment (tadaspetra) [inline INTRODUCTION.md:103]: Add the SHA-1 stuff :)

**Review by tadaspetra — CHANGES_REQUESTED** [review]

**Comment by GaithAST** [comment]
will fix rn

**Pull request closed without merging**